.github/workflows/ci.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         os: ${{ matrix.os }}
       env:
         CHEF_LICENSE: accept-no-persist
-        CHEF_VERSION: 16.18.0
+        CHEF_VERSION: 17.10.163
```

Comment:
Mind changing that to something more recent, say 18.5.0?

Reply:
Done